src/linear_algebra/multilinear/basic.lean

```diff
@@ -1177,3 +1186,93 @@ f.map (λ i, ⊤)
 end multilinear_map
 
 end submodule
+
+section basis
+
+open multilinear_map
+
+variables [comm_semiring R] [add_comm_monoid M₂] [add_comm_monoid M₃] [∀i, add_comm_monoid (M i)]
+variables [∀i, module R (M i)] [module R M₂] [module R M₃]
+
+/-- Two multilinear maps indexed by `fin n.succ` are equal if they are equal when all arguments
+are basis vectors. -/
+lemma basis.ext_multilinear_fin {f g : multilinear_map R M M₂} {ι₁ : fin n.succ → Type*}
+  {e : Π i, basis (ι₁ i) R (M i)} (h : ∀ i : Π j, ι₁ j, f (λ x, e x (i x)) = g (λ x, e x (i x))) :
```

Comment:
These indices are a little hard to follow because `i` has different types between the two. Perhaps
```suggestion
  {e : Π i, basis (ι₁ i) R (M i)} (h : ∀ v : Π i, ι₁ i, f (λ i, e i (v i)) = g (λ i, e i (v i))) :
```

Reply:
Also, I think `e` should probably be explicit, the chance of the elaborator inferring it from `h` seems pretty low.

Reply:
I've made it explicit (thereby avoiding the need for a `have` with explicit type specified to construct the `h` for the use of this lemma).